src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -88,60 +88,62 @@ public class C
       ""results"": [
         {{
           ""ruleId"": ""CS0169"",
-          ""kind"": ""warning"",
+          ""level"": ""warning"",
+          ""message"": ""The field 'C.x' is never used"",
           ""locations"": [
             {{
-              ""analysisTarget"": [
-                {{
-                  ""uri"": ""{0}"",
-                  ""region"": {{
-                    ""startLine"": 4,
-                    ""startColumn"": 17,
-                    ""endLine"": 4,
-                    ""endColumn"": 18
-                  }}
+              ""resultFile"": {{
+                ""uri"": ""{0}"",
+                ""region"": {{
+                  ""startLine"": 4,
+                  ""startColumn"": 17,
+                  ""endLine"": 4,
+                  ""endColumn"": 18
                 }}
-              ]
+              }}
             }}
           ],
-          ""fullMessage"": ""The field 'C.x' is never used"",
-          ""isSuppressedInSource"": false,
-          ""tags"": [
-            ""Compiler"",
-            ""Telemetry""
-          ],
           ""properties"": {{
-            ""severity"": ""Warning"",
-            ""warningLevel"": ""3"",
-            ""defaultSeverity"": ""Warning"",
```

Comment:
For example, a rule with defaultSeverity = Warning but actual serverity = Error tells me that the build has /warnaserror or a specific ruleset increasing the severity. Isn't this useful information?